Gemfile

```diff
@@ -0,0 +1,74 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.1.3'
```

Comment:
- [ ] Please make sure that you are consistent with the versions of your project, at the moment, you're having `ruby 3.1.3` specified in your Gemfile but the `.ruby-version` file specifies `ruby 3.2.1`. 🤓 🔧